src/test/java/org/fogbowcloud/manager/core/plugins/behavior/federationidentity/ldap/LdapLocalIdentityPluginTest.java

```diff
@@ -23,121 +25,83 @@
     private final String MOCK_SIGNATURE = "mock_signature";
 
     private LdapIdentityPlugin ldapStoneIdentity;
-
+    private final String name = "ldapUser";
+    private final String password = "ldapUserPass";
+    private final String userName = "User Full Name";
+    Map<String, String> userCredentials = new HashMap<String, String>();
+    
     @Before
     public void setUp() throws Exception {
-
         HomeDir.getInstance().setPath("src/test/resources/private");
         PropertiesHolder propertiesHolder = PropertiesHolder.getInstance();
         Properties properties = propertiesHolder.getProperties();
-        properties.put(IDENTITY_URL_KEY, KEYSTONE_URL);
-
+        properties.put(IDENTITY_URL_KEY, this.KEYSTONE_URL);
         this.ldapStoneIdentity = Mockito.spy(new LdapIdentityPlugin());
-        Mockito.doReturn(MOCK_SIGNATURE)
-                .when(ldapStoneIdentity)
+        Mockito.doReturn(this.MOCK_SIGNATURE)
+                .when(this.ldapStoneIdentity)
                 .createSignature(Mockito.any(JSONObject.class));
-    }
 
+        this.userCredentials.put(LdapIdentityPlugin.CRED_USERNAME, this.name);
+        this.userCredentials.put(LdapIdentityPlugin.CRED_PASSWORD, this.password);
+        this.userCredentials.put(LdapIdentityPlugin.CRED_AUTH_URL, "ldapUrl");
+        this.userCredentials.put(LdapIdentityPlugin.CRED_LDAP_BASE, "ldapBase");
+        this.userCredentials.put(LdapIdentityPlugin.CRED_LDAP_ENCRYPT, "");
+        this.userCredentials.put(LdapIdentityPlugin.CRED_PRIVATE_KEY, "private_key_path");
+        this.userCredentials.put(LdapIdentityPlugin.CRED_PUBLIC_KEY, "public_key_path");
+    }
+    
+    //test case: check if the access id information is correct when the user credentials are correct.
```

Comment:
I think it can be more clear, like: ... information is correct when creating a token with the correct user credentials.

Reply:
Great point, thank you